composer.json

```diff
@@ -23,7 +23,7 @@
     ],
     "require": {
         "php": ">=7.2|^8.0",
```

Comment:
I think You should  add php 8.1 
 ```
 "php": ">=7.2|^8.02|^8.1",
 ```

Reply:
Yes, agreed, merging this will also need to allow PHP 8.1 so it can be used with Laravel 10, which will require PHP 8.1.

Reply:
There is no nee do update the PHP version constraint. PHP 8.1 is already allowed. For reference see the [composer docs](https://getcomposer.org/doc/articles/versions.md#caret-version-range-).